autoprofile.py

```diff
@@ -33,32 +42,105 @@ def __init__(self):
         self.name = _("Automatic Profile")
         self.bio_enabled = False
         self.name_enabled = False
+        self.pfp_enabled = False
```

Comment:
Why is this (and the next one) instance properties when they could be locals?

Reply:
how else would I access this property from stopautopfp cmd?

Reply:
You don't need to. Just leave them be, if they are instance variables they are destroyed on function exit